recipes/deployment.rb

```diff
@@ -11,13 +11,13 @@
   if prefer :database, 'sqlite'
     gsub_file 'Gemfile', /.*gem 'sqlite3'\n/, ''
     add_gem 'sqlite3', group: [:development, :test]
-    add_gem 'pg', group: :production
+    add_gem 'pg', '>= 0.18', group: :production
   end
 end
 
 if prefer :deployment, 'capistrano3'
   say_wizard "installing gems for Capistrano"
-  add_gem 'capistrano', '~> 3.0.1', group: :development
+  add_gem 'capistrano', '>= 3.6.0', group: :development
```

Comment:
@MProG :)
`>= 3.6.0`
^ would indicate that __any__ version later than (or equal to) `3.6.0` is acceptable. There is no upper bound. 

So, gem will be updated to `4.x.x` while `bundle update` right after this version will be realized

We should update `capistrano` to major/minor versions by hands.